winterfell/controllers/authController.js

```diff
@@ -1,18 +1,17 @@
 var _ = require('lodash');
-var passport = require('passport');
 var constants = require('./../utils/constants');
 var http = require('http-status-codes');
 var httpErrors = require('./../utils/httpErrors');
+var Log = require('./../utils/logHelper');
+var passport = require('passport');
 var User = require('./../models/user');
 var UserValues = require('./../models/userValues');
 var UserService = require('./../services/userService');
 
 module.exports = function (app) {
 
   // Endpoint for routing sign-up
-  app.get('/signup', function(req, res) {
-    console.log('[signup] request received for ' + JSON.stringify(req.body));
-    console.log('[signup] session for ' + JSON.stringify(req.session));
+  app.get('/signup', [Log.api], function(req, res) {
```

Comment:
If you remove this manually added middlewares, I have a way to add a single middleware at app initialization and will log all the requests (using the `morgan` library).